apps/backend/api/src/entities/bonus-malus.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { Length } from 'class-validator';
+import { Column, Entity, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { Character } from './character.entity';
+import { Event } from './event.entity';
+
+@Entity()
+export class BonusMalus {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  type: string;
+
+  @Column()
+  @Length(30)
```

Comment:
Pas comme ça, le `@Length` sert pour la validation, pas SQL
```suggestion
  @Column('varchar', { length: 30 })
```

Reply:
Fait